reflectivity_ui/interfaces/data_handling/data_set.py

```diff
@@ -174,10 +174,19 @@ def set_parameter(self, param, value):
             logging.error("Could not set parameter %s %s", param, value)
         return has_changed
 
-    def calculate_reflectivity(self, direct_beam=None, configuration=None):
+    def calculate_reflectivity(self, direct_beam=None, configuration=None, ws_suffix: str = ""):
         """
         Loop through the cross-section data sets and update
         the reflectivity.
+
+        Parameters
+        ----------
+        direct_beam: CrossSectionData | None
+            Direct beam data
+        configuration: Configuration | None
+            The configuration
+        ws_suffix: str
+            String to add to reflectivity workspace name
```

Comment:
can you include a usage example (for instance, what would `ws_suffix` be when dealing with multiple ROI's)